test/unit/api_spec.js

```diff
@@ -2219,6 +2219,22 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       await loadingTask.destroy();
     });
 
+    it("gets text content, and check that out-of-page text is not present (bug 1755201)", async function () {
+      if (isNodeJS) {
+        pending("Linked test-cases are not supported in Node.js.");
+      }
+
+      const loadingTask = getDocument(buildGetDocumentParams("bug1755201.pdf"));
+      const pdfDoc = await loadingTask.promise;
+      const pdfPage = await pdfDoc.getPage(6);
+      const { items } = await pdfPage.getTextContent();
+      const text = mergeText(items);
+
+      expect([...text.matchAll(/win aisle/g)].length).toEqual(0);
```

Comment:
Could this not be simplified a bit, perhaps something like this:
```javascript
expect(/win aisle/.test(text)).toEqual(false);
```

Reply:
Another reason for changing this is that `matchAll` is new enough functionality that it'd require adding a polyfill...